### ISSUE TITLE
question about hardware-aware training example

### ISSUE THREAD

**Issue by saba-er** [IBM/aihwkit#247]
Hi,
Reading example 06_lenet5_hardware_aware and trying it, it does not seem to have implemented a fine tuning step after programming the weights. Is it not a necessary step in hardware-aware training? Also, why the noise model is bound to PCMLikeNoise? Are there other noise models supported?

Thanks!

**Comment by maljoras** [comment]
Hi @SaBaKa2020,
Many thank for your question.  We assume for the hardware-aware training and inference that the hardware is not capable of doing on-chip back propagation (no backward pass or update pass implemented). We thus indeed do not perform any fine-tuning except drift compensation, which adjusts the digital output scalings.   

If one had batch normalization in the DNN, it might be worthwhile to tune those statistics after programming. This can be easily added by the user, though.

**Comment by maljoras** [comment]
Regarding the noise, we have only a statistical model for the PCM noise so far, which we have measured on real hardware.  One can easily add other user-defined noise models by overriding the `BaseNoiseModel` (from `aihwkit.simulator.noise_models`).
